src/assignment/assignment.service.ts

```diff
@@ -1480,10 +1481,15 @@ export class AssignmentService {
 	}
 
 	static async addContributor(
-		assignmentId: string,
-		user: Partial<ContributorInfo>
+		assignment: Avo.Assignment.Assignment | null | undefined,
+		invitee: Partial<ContributorInfo>,
+		inviter?: Avo.User.CommonUser
 	): Promise<void> {
-		if (isNil(user.email) || isEmpty(user.email)) {
+		if (!assignment) return;
+
+		const assignmentId = assignment.id;
+
+		if (isNil(invitee.email) || isEmpty(invitee.email)) {
```

Comment:
!variable does the same as combining isNil and isEmpty and is a lot more efficient
```suggestion
		if (!invitee.email) {
```